shared-package.props

```diff
@@ -46,12 +46,12 @@
     <Error Text="The 'Description' property must be set for packable project '$(MSBuildProjectName)'." Condition="'$(Description)' == ''" />
   </Target>
 
-  <ItemGroup Condition="$(MSBuildProjectName.StartsWith('Steeltoe.Configuration'))">
+  <ItemGroup Condition="$(MSBuildProjectName.StartsWith('Steeltoe.Configuration')) Or $(MSBuildProjectName.StartsWith('Steeltoe.Management'))">
     <!-- Widen the condition above as we're completing more public API reviews -->
     <PackageReference Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="$(PublicApiAnalyzersVersion)" PrivateAssets="All" />
   </ItemGroup>
 
-  <PropertyGroup Condition="!$(MSBuildProjectName.StartsWith('Steeltoe.Configuration'))">
+  <PropertyGroup Condition="!$(MSBuildProjectName.StartsWith('Steeltoe.Configuration')) Or $(MSBuildProjectName.StartsWith('Steeltoe.Management'))">
```

Comment:
Note the `!` in the condition on the left. This should be expanded with "And !$..." instead of "Or $...".